apis/python/src/tiledbsoma/__init__.py

```diff
@@ -156,6 +156,7 @@
     get_SOMA_version,
     get_storage_engine,
     show_package_versions,
+    verify_core_versions,
```

Comment:
I would be happier with this being `_verify_core_versions` and not exported. This doesn't need to be public API. (Or rather, keep it `verify_core_versions` as it is in `_general_utilities` but then just don't auto-export it.)